source/common/http/conn_manager_impl.h

```diff
@@ -461,6 +465,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
   bool clear_hop_by_hop_response_headers_{true};
   // The number of requests accumulated on the current connection.
   uint64_t accumulated_requests_{};
+  bool use_proactive_draining_;
```

Comment:
nit: const

Reply:
done in https://github.com/envoyproxy/envoy/pull/16201/commits/80aed8cbe5b9a3132befdbcdc3ffb8fe865f3f86